src/clocksi_vnode.erl

```diff
@@ -125,13 +125,13 @@ abort(ListofNodes, TxId) ->
 init([Partition]) ->
     PreparedTx = ets:new(list_to_atom(atom_to_list(prepared_tx) ++
                                           integer_to_list(Partition)),
-                         [set, {write_concurrency, true}]),
+                         [set]),
     CommittedTx = ets:new(list_to_atom(atom_to_list(committed_tx) ++
                                            integer_to_list(Partition)),
-                          [set, {write_concurrency, true}]),
+                          [set]),
     ActiveTxsPerKey = ets:new(list_to_atom(atom_to_list(active_txs_per_key)
                                            ++ integer_to_list(Partition)),
-                              [bag, {write_concurrency, true}]),
+                              [bag]),
```

Comment:
That'd be a good thing to mention in the commit message, otherwise someone may think that they previously were being access by multiple processes. :)